6.3.22/index1.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css">
+	<title>Document</title>
+</head>
+<body>
+	<h1>Test 06/03/22</h1>
+	<form>
+		<buton>Click</buton>
+	</form>
+	<figure>
+		<figcaption></figcaption>
+	</figure>
+	<div id="container">
+	<div class="display"></div>
+	<div class="controls"></div>
+</div>
+
+	<button id="btn">Click please</button>
+
+
+<select id="selcolor">
+   <option value="Maroon">Red</option>
+	<option value="Green">Green</option>
+</select>
+<button onclick="clickPlease()">button</button>
+<div id="pict"></div>
+
+
+
+
+
+	<script type="script.js"></script>
+</body>
+</html>
```

Comment:
@VladSkorokhoda it can be attempting to use HTML, but the test asked to use only JS to complete the task.
You can use javascript document.createElement to create an element, `.append` to add it to the body for instance and so on.